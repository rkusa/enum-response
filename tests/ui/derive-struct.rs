#[macro_use]
extern crate api_error_derive;

#[derive(ErrorStatus)]
struct Test;

