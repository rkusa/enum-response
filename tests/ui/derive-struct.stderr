error: proc-macro derive panicked
 --> $DIR/derive-struct.rs:4:10
  |
4 | #[derive(ErrorStatus)]
  |          ^^^^^^^^^^^
  |
  = help: message: #[derive(EnumFrom)] can only be applied to enums. Test is not an enum.

